server/controllers/api.py

```diff
@@ -811,6 +824,7 @@ def get(self, user, email=None):
 
 # Other
 api.add_resource(Enrollment, '/v3/enrollment/<string:email>/')
+api.add_resource(CourseEnrollment, '/v3/course_enrollment/<string:cid>')
```

Comment:
- cid should be an int right? Regardless - We're trying to do away with using course IDs (since no-one knows what the ID is off the top of their heads - and go by the course offering 

> '/v3/course_enrollment/<offering:offering>'